.travis.yml

```diff
@@ -83,7 +83,7 @@ before_install: |
   fi
 
 install:
-  - pip install numpy==1.8.0 six twine pytest-cov coveralls
+  - pip install numpy==1.10.4 pyarrow==0.4.0 six twine pytest-cov coveralls
```

Comment:
This is the minimal numpy requirement for Arrow & Python 3 due to some bugs in NumPy Macros. We can stay on a lower version for Python 2 if needed. But as e.g. conda packages are nowadays built against 1.11+, it would probably be ok to drop some older versions of NumPy now.